pydoc/getting_started/tutorial/example.md

```diff
@@ -0,0 +1,151 @@
+# Example
+
+Let's say that you have the following Python class:
+
+```python
+class Person:
+    """A person.
+
+    Arguments:
+        name: Full name.
+        age: Age of person.
+        skills: List of skills.
+
+    """
+
+    def __init__(self, name: str, age: float, skills: list[str]) -> None:
+        self.name = name
+        self.age = age
+        self.skills = skills
+```
+
+You want to describe this class semantically.
+This is done by defining the following metadata (using JSON) identifying the Python attributes with DLite properties.
+Here we define `name` to be a string, `age` to be a float and `skills` to be an array of `N` strings, where `N` is a name of a dimension.
+The metadata uniquely identifies itself with the "name", "version" and "namespace" fields and "meta" refers to the metadata schema (meta-metadata) that this metadata is described by.
+Finally, human descriptions of the metadata itself, its dimensions and its properties are provided in the "description" fields.
+
+```json
+{
+  "name": "Person",
+  "version": "0.1",
+  "namespace": "http://onto-ns.com/meta",
+  "meta": "http://onto-ns.com/meta/0.3/EntitySchema",
+  "description": "A person.",
+  "dimensions": [
+    {
+      "name": "N",
+      "description": "Number of skills."
+    }
+  ],
+  "properties": [
+    {
+      "name": "name",
+      "type": "string",
+      "description": "Full name."
+    },
+    {
+      "name": "age",
+      "type": "float",
+      "unit": "year",
+      "description": "Age of person."
+    },
+    {
+      "name": "skills",
+      "type": "string",
+      "shape": ["N"],
+      "description": "List of skills."
+    }
+  ]
+}
+```
+
+The metadata can be saved in file ``Person.json``.
+In Python we can now make a DLite-aware subclass of `Person`, instantiate it, and serialise it to a storage:
+
+```python
+import dlite
+
+# Create a DLite-aware subclass of Person
+DLitePerson = dlite.classfactory("Person", url="json://Person.json")
+
+# Instantiate
+person = DLitePerson(
+    "Sherlock Holmes",
+    34.,
+    ["observing", "chemistry", "violin", "boxing"],
+)
+
+# Write to storage (here a JSON file)
+person.dlite_inst.save('json://homes.json?mode=w')
+```
+
+To access this new instance from C, you can first generate a header file from the meta data
```

Comment:
This belongs to doc/code-generation.md

Reply:
Sure. Again, a great issue for a future PR. The purpose of _this_ PR is to split up the README and place it (hopefully not too temporarily) under the `pydoc/` (future `doc/`) folder.